### ISSUE TITLE
[Arcs] Tax limit needs to be ignored

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#107]
[Tax limit](https://arcs.seiyria.com/#7.1.2-tax-limit):

> You cannot tax a city more than once per turn.

We know that [Cannot is absolute](https://arcs.seiyria.com/#22.4.2-cards). So technically cards that allow for breaking this limit need to explicitly say "**ignore** tax limit".

- [Warlord's Cruelty](https://cards.ledergames.com/card/ARCS-L23)
- [Overseer](https://cards.ledergames.com/card/ARCS-LEAD10)

**Comment by jdyearsley** [comment]
Yeah, I dunno. I get the argument from a legalistic POV, but these directly say you do it again. Definitely not worth errata, which adds confusion on its own. But I'll add FAQs.

**Comment by jdyearsley** [comment]
I actually don't even think this is FAQ-worthy. The text says what it says, so I'm inclined to leave everything as-is unless an example comes up of someone actually being confused by the intent here.
